library/std/src/os/windows/process.rs

```diff
@@ -181,6 +181,13 @@ pub trait CommandExt: Sealed {
     #[stable(feature = "windows_process_extensions", since = "1.16.0")]
     fn creation_flags(&mut self, flags: u32) -> &mut process::Command;
 
+    /// Sets the field [wShowWindow][1] of [STARTUPINFO][2] that is passed to `CreateProcess`.
+    ///
+    /// [1]: https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-showwindow
```

Comment:
At first I thought this was just the wrong link since it doesn't mention `wShowWindow` at all, but after reading both links I see how they are connected. It's not what the sentence above implies.

I think it would be clearer to drop the link on `wShowWindow` in this sentence and link only the `STARTUPINFO` (second link). Then, since that link doesn't supply the table of possible values, in a second sentence we can say the allowed values are the ones listed in [link to ShowWindow].

Reply:
Done! I hope I got that right in my last commit :)